Northstar.Client/mod/scripts/vscripts/_custom_codecallbacks_client.gnut

```diff
@@ -23,7 +23,7 @@ struct {
 
 void function OnReceivedMessage(ClClient_MessageStruct localMessage) {
 
-    if ( IsWatchingReplay() )
+    if ( IsWatchingReplay() && ( GetLocalClientPlayer() != localMessage.player ) )
```

Comment:
```suggestion
    if ( IsWatchingReplay() && localMessage.player == null )
```
as discussed in discord, this would allow for duplicate client messages (if that was even a problem before) but not show server messages in replays, meaning no duplicate server messages

this is still a bandaid fix, but should be a bit better than current

Reply:
do `!(localMessage.player)`

Reply:
nah, rest of the file uses `localMessage.player == null` to indicate server, so this way improves readability. If you want to change this, open a PR and change it everywhere in the file

Reply:
yeah this looks better than my thing